core/src/main/scala/tapir/Api.scala

```diff
@@ -0,0 +1,24 @@
+package tapir
+import tapir.Api.Info
+
+case class Api(info: Info)
+
+object Api {
+  case class Info(
+      title: String,
+      version: String,
+      description: Option[String] = None,
+      termsOfService: Option[URL] = None,
+      contact: Option[Contact] = None,
+      license: Option[License] = None
+  )
+  case class Contact(name: Option[String], email: Option[Email], url: Option[URL])
+  case class License(name: String, url: Option[URL])
+
+  def apply(title: String, version: String): Api = {
+    new Api(info = Info(title, version))
+  }
+
+  type Email = String
+  type URL = String
```

Comment:
I need here and in some other places and URI class with parse. Should I just copy Uri form sttp code for now or use java.net.URI?

Reply:
I think `java.net.URI` should be sufficient. The main benefit of the sttp one is building URIs through interpolation, which we don't really need here